windows-driver-docs-pr/mobilebroadband/cosa-overview.md

```diff
@@ -16,7 +16,7 @@ Here are some frequently asked questions about COSA:
 
 ## What are the settings that mobile operators can specify in COSA?
 
-The settings are largely the same as what MOs configured in apndatabase.xml, with a few exceptions and new additions. For details, see the tables in [Planning your desktop COSA database submission](planning-your-desktop-cosa-database-submission.md).
+The settings are largely the same as what MOs configured in apndatabase.xml, with a few exceptions and new additions. For details, see the tables in [Desktop COSA database settings](desktop-cosa-database-settings.md).
```

Comment:
This APN database stuff is so old. I don't think it's even worth keeping that reference here.

Reply:
I'll scrub for any references to it.